Doc/releasenote.md

```diff
@@ -3,6 +3,12 @@
 * Add three algorithms for constrained optimization (i.e., CCMO, MOEA/D-DAE, and TiGE-2) and an algorithm for many-objective optimization (i.e., PREA). There are currently 118 algorithms in the platform.
```

Comment:
The releasenote is used to record the historical version notes. so, it would be better to add a new note when another new is added.